tests/test_pubsub.py

```diff
@@ -474,3 +475,31 @@ def t_run(rc):
             t.start()
     except Exception:
         print("Error: unable to start thread")
+
+
+class TestPubSubPubSubSubcommands(object):
```

Comment:
It's the same name as the test case in Redis Py, so for symmetry I kept it -- The name is a bit of a mouthful and made in jest, though it does Test the Pub/Sub system's PUBSUB subcommands, so it is descriptive